composer.json

```diff
@@ -11,13 +11,13 @@
         }
     ],
     "require" : {
-        "php": ">=5.4.0",
+        "php": "~5.5|~7.0",
         "guzzlehttp/guzzle": "~5.2",
-        "symfony/event-dispatcher": "~2.6"
+        "symfony/event-dispatcher": "~2.8|~3.0"
```

Comment:
It's not mandatory. But since Symfony3 is BC with Symfony2.8, we can keep it without much harm for those who are stuck with Symfony2.